tox.ini

```diff
@@ -18,34 +19,28 @@ setenv =
     PURE_PYTHON = 1
     PIP_CACHE_DIR = {envdir}/.cache
 
-[testenv:py33-pure]
+[testenv:py34-pure]
 basepython =
-    python3.3
+    python3.4
 setenv =
     PURE_PYTHON = 1
     PIP_CACHE_DIR = {envdir}/.cache
 
 [testenv:coverage]
+usedevelop = true
 basepython =
     python2.7
 commands =
-#   The installed version messes up nose's test discovery / coverage reporting
-#   So, we uninstall that from the environment, and then install the editable
-#   version, before running nosetests.
-    pip uninstall -y zope.security
-    python -c "import shutil; shutil.copyfile('src/zope/__init__.py', '{envdir}/lib/python2.7/site-packages/zope/__init__.py')"
-    pip install -e .
-    nosetests --with-xunit --with-xcoverage
+    coverage run -m zope.testrunner --test-path=src []
+    coverage run -a -m sphinx -b doctest -d {envdir}/.cache/doctrees docs {envdir}/.cache/doctest
+    coverage report --fail-under=92
```

Comment:
Can't we put the --fail-under in `.coveragerc`?

Do we want Travis to fail if coverage decreases?  (I'm mildly in favour.)

Reply:
coveralls.io has its own separate setting for failing if coverage decreases (and I currently have that on, set to the same thresholds). Since it doesn't get the source checkout (I think?) it can't read .coveragerc.